src/lib.rs

```diff
@@ -320,7 +330,9 @@ mod test {
         let blocking_client = builder.build_blocking();
 
         let builder_async = Builder::new(&format!("http://{}", esplora_url));
-        let async_client = builder_async.build_async().unwrap();
+        let async_client = builder_async
+            .build_async_with_sleeper::<DefaultSleeper>()
+            .unwrap();
```

Comment:
I'd use it as below instead, so we can make sure we're testing using the default `build_async` and `build_with_sleeper` in different feature matrix scenarios.

``` suggestion
        #[cfg(feature = "tokio")]
        let async_client = builder_async.build_async().unwrap();

        #[cfg(not(feature = "tokio"))]
        let async_client = builder_async
            .build_async_with_sleeper::<r#async::DefaultSleeper>()
            .unwrap();
```